www/js/index.js

```diff
@@ -136,6 +136,12 @@ function findLine(e) {
     lineFound.click();
   }
 }
+function feelingLucky() {
+  if (search.$results.childNodes.length === 1) {
```

Comment:
i wouldn't call this a "feeling lucky". feeling lucky would be if `search.$results.childNodes.length` is `>=1`